MODULE.bazel

```diff
@@ -24,14 +24,25 @@ module(
 bazel_dep(name = "rules_proto", version = "7.0.2")
 bazel_dep(name = "rules_python", version = "0.37.0")
 bazel_dep(name = "platforms", version = "0.0.10")
-bazel_dep(name = "protobuf", version = "28.3")
+# TF proto version is protobuf-3.21.9, and per proto engdoc
+# https://protobuf.dev/support/version-support/
+# The major version saparation happens at 21.x == 3.21
+# so we pick 21.7, which is the same generation as 3.21.9
+bazel_dep(name = "protobuf", version = "21.7")
```

Comment:
Hi @dryman!

With @iindyk we purposefully upgraded protobuf version (we need it for conda-forge feedstocks). The good news is that a few days ago tensorflow 2.20.0rc1 was released with `protobuf-6.31.1`. We now use this TF version for testing.